assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -42,5 +42,12 @@ export default function useExistingTagEffect() {
 				await selectContainerByID( existingTag );
 			}
 		} )();
-	}, [ hasExistingTag, existingTag, hasExistingTagPermission, existingTagPermission ] );
+	},	[
+		hasExistingTag,
+		existingTag,
+		hasExistingTagPermission,
+		existingTagPermission,
+		selectAccount,
+		selectContainerByID,
+	]	);
```

Comment:
What's with the indentation of the `[` and `]` here?

```suggestion
	}, [
		hasExistingTag,
		existingTag,
		hasExistingTagPermission,
		existingTagPermission,
		selectAccount,
		selectContainerByID,
	] );
```